analysis_options.yaml

```diff
@@ -10,29 +11,69 @@ linter:
   rules:
     - annotate_overrides
     - avoid_empty_else
+    - avoid_function_literals_in_foreach_calls
     - avoid_init_to_null
+    - avoid_null_checks_in_equality_operators
+    - avoid_relative_lib_imports
     - avoid_return_types_on_setters
+    - avoid_returning_null
+    - avoid_types_as_parameter_names
+    - avoid_unused_constructor_parameters
     - await_only_futures
     - camel_case_types
+    - cancel_subscriptions
     - comment_references
+    #- constant_identifier_names
     - control_flow_in_finally
     - directives_ordering
     - empty_catches
     - empty_constructor_bodies
     - empty_statements
     - hash_and_equals
     - implementation_imports
+    - invariant_booleans
+    - iterable_contains_unrelated_type
     - library_names
     - library_prefixes
+    - list_remove_unrelated_type
+    - no_adjacent_strings_in_list
+    - no_duplicate_case_values
```

Comment:
Could you remove all the `pedantic` dups?

```
    - no_duplicate_case_values
    - prefer_contains
    - prefer_equal_for_default_values
    - prefer_is_not_empty
    - recursive_getters
    - unrelated_type_equality_checks
    - use_rethrow_when_possible
    - unawaited_futures
    - valid_regexps
```

See: https://github.com/dart-lang/pedantic/blob/v1.3.0/lib/analysis_options.yaml